articles/quickstart/native/ios-swift/08-touch-id-authentication.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Touch ID Authentication
+description: This tutorial will show you how to utilise Touch ID and the Credentials Manager
+budicon: 345
+---
+
+<%= include('../../../_includes/_package', {
+  org: 'auth0-samples',
+  repo: 'auth0-ios-swift-sample',
+  path: '00-Credentials-TouchID',
+  requirements: [
+    'CocoaPods 1.2.1',
+    'Version 8.3.2 (8E2002)',
+    'iPhone 7 - iOS 10.3 (14E269)',
+    'Touch ID Enrolled Device'
+  ]
+}) %>
+
+## Before Starting
+
+You should be familiar with previous tutorials. This tutorial assumes that you've integrated the [Auth0.swift](https://github.com/auth0/Auth0.swift/) dependencies in your project and you're familiar with presenting the Login. For further information, check out the [Login](/quickstart/native/ios-swift/00-login) quickstart.
+It is recommended that you are explore the [User Sessions](/quickstart/native/ios-swift/03-user-sessions) quickstart if you are not familiar with renewal of credentials using a `refreshToken`.
+
+## Touch ID Authentication
+
+Here's the scenario: You are using `webAuth` to present the HLP for the user to Login. After user authentication you want to store the user's credentials and use the `refreshToken` to renew the user's credentials without having to present the HLP. Additionally you want to utilize Touch ID to validate this renewal process.
```

Comment:
Do you mention `HLP` somewhere before? If not, explain or use the large version `"Hosted Login Page" (HLP)`